requirements.txt

```diff
@@ -5,6 +5,7 @@ docutils==0.16
 ebaysdk==2.1.5
 freezegun==0.3.11; python_version < '3.8'
 freezegun==0.3.15; python_version >= '3.8'
+geoip2==2.9.0
```

Comment:
changes to requirements require checks with @d-fence and @antonylesuisse, so all the relevant people are notified in advance. Was this done elsewhere perhaps?

Reply:
d-fense is in front of me in the office, I've noticed him orally

Reply:
I'll ping the relevant people right now.

Reply:
10 days later, are we all set ?

Reply:
5 months later, are we all set?

Reply:
Please rebase.